index.ts

```diff
@@ -49,6 +44,13 @@ export function withHookdeck(config: HookdeckConfig, f: Function): (args) => Pro
         return Promise.resolve(f.apply(this, args));
       }
 
+      // Check if vercel or next env is develoment
+      const is_development = process && (process.env.VERCEL_ENV === 'development' || process.env.NODE_ENV === 'development');
+      if (is_development) {
+        console.warn('[Hookdeck] Detected local development enviroment. Cannot forward to Hookdeck. Bypassing to middleware...');
```

Comment:
[Hookdeck] Local development environment detected. Hookdeck middleware is disabled locally. Bypassing the middleware...

Reply:
changed